dist/node/admin.js

```diff
@@ -501,8 +501,14 @@ var StitchAdminClient = exports.StitchAdminClient = function (_StitchClient) {
                 confirmByEmail: function confirmByEmail(email) {
                   return api._post(appUrl + '/user_registrations/by_email/' + email + '/confirm');
                 },
-                listUnconfirmed: function listUnconfirmed(filter) {
-                  return api._get(appUrl + '/user_registrations/unconfirmed_users', filter);
+                listPending: function listPending(filter) {
```

Comment:
i think you need to rebuild and then push, this still says `listUnconfirmed` from before, which is incorrect

Reply:
the diff seems to say listPending replaces listUnconfirmed in this file now?

Reply:
i guess someone forgot to rebuild